group_vars/maintenance.yml

```diff
@@ -345,6 +345,13 @@ telegraf_plugins_extra:
       - timeout = "10s"
       - data_format = "influx"
       - interval = "24h"
+  galaxy_wizard_openai_utilization_and_costs:
+    plugin: "exec"
+    config:
+      - commands = ["{{ custom_telegraf_env }} OPENAI_API_KEY_TELEGRAF={{ openai_api_key_telegraf }} /usr/bin/galaxy_wizard_openai_utilization_and_costs.py"]
```

Comment:
is this run as telegraf user or something? If so we could populate the API key in the bashrc of this user and just provide here the ENV var?

Reply:
I have added a new post task to add the key to the telegraf user's bashrc and then updated the telegraf command to start a new shell and to source the bashrc file. Please re-review the PR.